src/util/index.test.ts

```diff
@@ -240,7 +250,7 @@ describe('attr', () => {
   it('gets attributes of an element', () => {
     const elem = create('<img foo="bar">')
     expect(attr(elem, 'foo')).toEqual('bar')
-    expect(attr(elem, 'baz')).toEqual(null)
+    expect(attr(elem, 'baz')).toEqual(undefined)
```

Comment:
Returning `null` rather than `undefined` is what the DOM does when requested a non-existent attribute. Worth keeping the same behaviour here?